script/brave_license_helper.py

```diff
@@ -38,6 +38,34 @@ def AddBraveCredits(root, prune_paths, special_cases, prune_dirs,
         os.path.join('brave', 'third_party', 'libaddressinput'),
         os.path.join('brave', 'patches', 'third_party'),
 
+        # Duplicate notices already included in brave-core notices
```

Comment:
This comment confused me at first. Can we rephrase it to:
```suggestion
        # Dependencies that are already in brave-core, and whose notices
        # therefore do not need to be repeated.
```